integrationTests/testProcessorNode.go

```diff
@@ -1381,24 +1382,26 @@ func (tpn *TestProcessorNode) initInnerProcessors(gasMap map[string]map[string]u
 		tpn.BlockTracker,
 		TestBlockSizeComputationHandler,
 		TestBalanceComputationHandler,
+		tpn.EpochNotifier,
+		tpn.ScheduledMiniBlocksEnableEpoch,
 	)
 	tpn.PreProcessorsContainer, _ = fact.Create()
 
 	argsTransactionCoordinator := coordinator.ArgTransactionCoordinator{
-		Hasher:               TestHasher,
-		Marshalizer:          TestMarshalizer,
-		ShardCoordinator:     tpn.ShardCoordinator,
-		Accounts:             tpn.AccntState,
-		MiniBlockPool:        tpn.DataPool.MiniBlocks(),
-		RequestHandler:       tpn.RequestHandler,
-		PreProcessors:        tpn.PreProcessorsContainer,
-		InterProcessors:      tpn.InterimProcContainer,
-		GasHandler:           tpn.GasHandler,
-		FeeHandler:           tpn.FeeAccumulator,
-		BlockSizeComputation: TestBlockSizeComputationHandler,
-		BalanceComputation:   TestBalanceComputationHandler,
-		EconomicsFee: tpn.EconomicsData,
-		TxTypeHandler: txTypeHandler,
+		Hasher:                            TestHasher,
+		Marshalizer:                       TestMarshalizer,
+		ShardCoordinator:                  tpn.ShardCoordinator,
+		Accounts:                          tpn.AccntState,
+		MiniBlockPool:                     tpn.DataPool.MiniBlocks(),
+		RequestHandler:                    tpn.RequestHandler,
+		PreProcessors:                     tpn.PreProcessorsContainer,
+		InterProcessors:                   tpn.InterimProcContainer,
+		GasHandler:                        tpn.GasHandler,
+		FeeHandler:                        tpn.FeeAccumulator,
+		BlockSizeComputation:              TestBlockSizeComputationHandler,
```

Comment:
too many spaces.

Reply:
It is correct if you look below at line 1405 you find this line:
BlockGasAndFeesReCheckEnableEpoch: tpn.BlockGasAndFeesReCheckEnableEpoch,